lisa/testsuite.py

```diff
@@ -290,7 +290,18 @@ def _send_result_message(self, stacktrace: Optional[str] = None) -> None:
             # force refresh information, when test result status is changed. The
             # refreshed information is not used so far. But in case it's needed
             # in future, keep it up to date.
-            self._environment_information = self.environment.get_information()
+            # avoid fetching information unnecessarily when rerun env
+            force_run = True
```

Comment:
The information maybe changed by install new kernel or other situation. So it needs to force run every time.